.github/config/android-arm.json

```diff
@@ -3,35 +3,8 @@
   "platforms": [
     "android-arm"
   ],
-  "test_on_device": true,
```

Comment:
Instead of removing all of this could we simply set test_on_device and test_e2e both to false for now?

(relevant for all instances like this)